EPRN.UnitTests/Portal/Services/WasteServiceTests.cs

```diff
@@ -21,12 +23,20 @@ public void Init()
         {
             _mockMapper = new Mock<IMapper>();
             _mockHttpWasteService = new Mock<IHttpWasteService>();
-
+            _mapper = GetMapper();
```

Comment:
The AutoMapper functionality (ie: profiles) doesn't need to be tested, as it will always resolve to the mapped object defined. It is, however, incumbent on the individual unit test to assert that the result is of the object type expected. This would be the case whether you are using a mocked mapper, or an actual one, and that is what the last line of my unit test is doing.

Reply:
It's not necessary for the unit test. And if we had more complex mappings it would most certainly need tests to confirm they've been done right. You can confirm that whatever has been passed _to_ the mapper interface is what you would expect. That's all you need to do. If there's processing required after that, then the mapper interface needs to be setup to return relevant data in the mocking.

Reply:
Done